x-pack/plugins/fleet/server/routes/epm/index.ts

```diff
@@ -130,15 +149,18 @@ export const registerRoutes = (routers: { rbac: FleetRouter; superuser: FleetRou
           maxBytes: MAX_FILE_SIZE_BYTES,
         },
       },
+      fleetRequireSuperuser: true,
```

Comment:
Similar to the setup privilege, I wonder if we should have a `fleetAuthz.fleet.uploadPackage` field and then calculate this in `calculateAuthz` (by passing in a `isSuperUser` argument to that function).